nRFCloud/device-simulator-v2
# feat: Adding in fake RSRP and Location service data

### Summary
When working with location service data on the FE we currently only have GPS/GNSS to test the location service map with. The goal of this is to add some points around Portland that send random AppId for location services [MCELL, SCELL, WIFI].

Additionally this add's RSRP support.

### Testing
1. Checkout this branch
2. Log into dev.nrfcloud.com
3. `yarn build`
4. `node dist/cli.js -k <API-key> -h https://api.dev.nrfcloud.com -a -d <DEVICE-ID> -s rsrp,location`
5. Go to the device view, should see points on the map with uncertainty circles

### Notes
* The BE isn't a fan of the appids MCELL and SCELL, so the device terminal will show some `50010` errors, not sure how to fix.



ticket: https://projecttools.nordicsemi.no/jira/browse/IRIS-4526
